src/govuk/components/exit-this-page/exit-this-page.mjs

```diff
@@ -117,10 +117,19 @@ ExitThisPage.prototype.exitPage = function (e) {
     redirectUrl = e.target.href
   }
 
+  this.$updateSpan.innerText = ''
+
   // Blank the page
+  // As well as creating an overlay with text, we also set the body to hidden
+  // to prevent screen reader and sequential navigation users potentially
+  // navigating through the page behind the overlay during loading
+  document.body.className = ((document.body.className) ? document.body.className + ' govuk-exit-this-page-hide-content' : 'govuk-exit-this-page-hide-content')
   this.$overlay = document.createElement('div')
   this.$overlay.className = 'govuk-exit-this-page__overlay'
```

Comment:
Do we want to change the name of this class to `govuk-exit-this-page-overlay`, as we seem to have started doing that for non-nested-but-related class names?

Reply:
Agree. I remember I requested we BEMify that name when you first implemented it but I acknowledge that was silly and we should be consistent with how we treat non-nested-but-related stuff so that we're following BEM properly. I'll sort it out.